pypiper/utils.py

```diff
@@ -670,7 +675,7 @@ def _add_args(parser, args, required):
     }
 
     if len(required) > 0:
-        requiredNamed = parser.add_argument_group('required named arguments')
+        required_named = parser.add_argument_group('required named arguments')
```

Comment:
I guess I've been writing too much R recently...

Reply:
Ha that's what I was thinking--I've caught myself doing this more lately, too,